app/cafe.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
```

Comment:
Code Style Issue: It's better to use parentheses when importing multiple classes/modules for better readability. Here is how it should look:

python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```